syne_tune/optimizer/schedulers/ask_tell_scheduler.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+from typing import Dict
+import datetime
+
+from syne_tune.backend.trial_status import Trial, Status, TrialResult
+from syne_tune.optimizer.scheduler import TrialScheduler
+
+
+class AskTellScheduler:
+    base_scheduler: TrialScheduler
+    trial_counter: int
+    completed_experiments: Dict[int, TrialResult]
+
+    def __init__(self, base_scheduler: TrialScheduler):
+        """
+        Simple interface to use SyneTune schedulers in a custom loop, for example:
```

Comment:
Not necessarily for now but perhaps we could add an example.

Reply:
there is already an example. I will update it in another PR